airflow/configuration.py

```diff
@@ -236,6 +236,16 @@ class AirflowConfigParser(ConfigParser):
         ("scheduler", "parsing_cleanup_interval"): ("scheduler", "deactivate_stale_dags_interval", "2.5.0"),
     }
 
+    # A mapping of new configurations to a list of old configurations for when one configuration
+    # deprecates more than one other deprecation.
```

Comment:
```suggestion
    # deprecates more than one other deprecation.
    # There is no special handling for values set under the old options - you must handle that yourself.
```

Or something like that. Let's be explicit.